internal/store/alert_configuration.go

```diff
@@ -112,3 +116,17 @@ func (s *Store) UpdateAlertConfiguration(alertConfig *atlas.AlertConfiguration)
 		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
 	}
 }
+
+// EnableAlertConfiguration encapsulate the logic to manage different cloud providers.
+func (s *Store) EnableAlertConfiguration(projectID, id string, enable bool) (*atlas.AlertConfiguration, error) {
```

Comment:
instead of passing a boolean could we have dedicated enable/disable dedicated methods?

Reply:
sure, but why?

Reply:
it's a bit confusing a method called Enabled which takes false to actually disable, not blocking in any way it's just making it a bit more consumable